### PR TITLE
Migrate EmbeddedContainer, Install Accordion

### PR DESCRIPTION
**Pull request by rkusan00**
This PR migrates EmbeddedContainer to add support for composite content elements, also installs first composite content element, ce-accordion

### CODE REVIEW

File: apps/frontend/plugins/embedded-container.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { EmbeddedContainer } from '@tailor-cms/core-components';
+
+export default defineNuxtPlugin((nuxt) => {
+  nuxt.vueApp.component('EmbeddedContainer', EmbeddedContainer);
```
Comment (underscope) [inline apps/frontend/plugins/embedded-container.ts:4]: Consider prefixing tailor components (e.g. TEmbeddedContainer or TailorEm...)

File: packages/core-components/package.json [hunk 2]
```diff
@@ -18,11 +18,13 @@
     "@tiptap/starter-kit": "^2.4.0",
     "@tiptap/vue-3": "^2.4.0",
     "@types/lodash": "^4.17.10",
+    "@types/uuid": "^10.0.0",
     "@types/pluralize": "^0.0.33",
     "date-fns": "^3.6.0",
     "lodash": "^4.17.21",
     "p-min-delay": "^4.0.2",
     "pluralize": "^8.0.0",
+    "uuid": "^10.0.0",
```
Comment (underscope) [inline packages/core-components/package.json:27]: Should be replaced with @paralleldrive/cuid2

File: packages/core-components/src/components/EmbeddedContainer.vue [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+const embeds = computed(() => {
+  const items = Object.values(props.container.embeds ?? {});
+  return sortBy(items, 'position') as ContentElement[];
+});
+
+const addItems = (items: ContentElement[]) => {
+  items = Array.isArray(items) ? items : [items];
+  const container = cloneDeep(props.container);
+  container.embeds = { ...container.embeds, ...mapKeys(items, 'id') };
```
Comment (underscope) [inline packages/core-components/src/components/EmbeddedContainer.vue:64]: Should we use uids (since we use that also to ref these elements in the UI)

Comment (rkusan00) [reply packages/core-components/src/components/EmbeddedContainer.vue:64]: ids in the embeds are actually uuids https://github.com/tailor-cms/author/blob/6912c856be8287d9902cd5f71f198d3559b10568/packages/core-components/src/components/AddElement/index.vue#L189

**Pull request merged**